pages/other/screens.mdx

```diff
@@ -0,0 +1,23 @@
+# Running Snailycad under a screen with linux!
+
+You can run snailycad under a screen so you can safely close the terminal/ssh window without risking it closing out on you!
+
+## Creating a screen
+
+First you need to create the screen do the following:
+`screen -S snailycad`
+You can change `snailycad` to whatever it's just a name nothing important
+
+after you enter it, it will immediatly open a screen session here you can goto your snailycad folder nd do 
+`yarn run concurrently "yarn workspace @snailycad/client start" "yarn workspace @snailycad/api generate && yarn workspace @snailycad/api start"`
```

Comment:
```suggestion
after you enter it, it will immediatly open a screen session here you can goto your snailycad folder and run:
 
```bash
yarn run concurrently "yarn workspace @snailycad/client start" "yarn workspace @snailycad/api generate && yarn workspace @snailycad/api start"
```
```